Latest version of onnxscript does not work for Python 3.8 `TypeError: 'type' object is not subscriptable`

```python
onnxscript/optimizer/_inliner.py:22:
    CallStack = list[CallSiteId]
E   TypeError: 'type' object is not subscriptable
```

`list[CallSiteId]` type annotation is only supported for Python 3.9+.

A fix would be to replace this with `typing.List[CallSiteId]`

--- a/onnxscript/optimizer/_inliner.py
+++ b/onnxscript/optimizer/_inliner.py
@@ -5,7 +5,7 @@
 from __future__ import annotations
 
 from collections import defaultdict
-from typing import Iterable, Sequence, Tuple
+from typing import Iterable, List, Sequence, Tuple
 
 import onnxscript.ir as ir
 import onnxscript.ir.convenience as ir_convenience
@@ -19,7 +19,7 @@ NodeReplacement = Tuple[Sequence[ir.Node], Sequence[ir.Value]]
 # outermost call site, and the last element is the innermost call site. This is used
 # primarily for generating unique names for values in the inlined functions.
 CallSiteId = str
-CallStack = list[CallSiteId]
+CallStack = List[CallSiteId]
 
 
 def _make_unique_name(name: str, callstack: CallStack, used_names: set[str]) -> str:

